base/src/com/google/idea/blaze/base/lang/buildfile/lexer/BuildLexerBase.java

```diff
@@ -769,10 +772,28 @@ private void tokenize() {
           popParen();
           break;
         case '>':
-          addToken(TokenKind.GREATER, pos - 1, pos);
+          if (lookaheadIs(0, '>') && lookaheadIs(1, '=')) {
```

Comment:
Maybe we can add a tokenize three chars and move the rest (<<, >>) also to two chars?

Reply:
I just copied the code which handles `//=`. Sure, this whole class could use some refactoring (personally I would prefer using something like ANTLR to generate the parser), but I want to keep my changes as simple and straightforward as possible.